platform/core/src/studies/services/wado/seriesLoadQueue.test.js

```diff
@@ -0,0 +1,96 @@
+import { api } from 'dicomweb-client';
+import { makeSeriesLoadQueue } from './seriesLoadQueue';
+
+describe('seriesLoadQueue.js', function() {
+  let seriesLoadQueue;
+  const studyId = '1.2.3.4';
+  const seriesIds = ['1.2.3.4.1', '1.2.3.4.2'];
+  const dicomWebClient = new api.DICOMwebClient();
+
+  beforeEach(function() {
+    seriesLoadQueue = makeSeriesLoadQueue(dicomWebClient, studyId, seriesIds);
+  });
+
+  it('should create a series load queue instance', function() {
```

Comment:
This particular test only checks if the properties defined on the object exist. I don't think this is useful test to have. 

The tests below that check for event emission and consumption work as intented are nice though.